example/src/test/java/com/capgemini/mrchecker/selenium/projectY/FileUploadTest.java

```diff
@@ -0,0 +1,64 @@
+package com.capgemini.mrchecker.selenium.projectY;
+
+import com.capgemini.mrchecker.core.groupTestCases.testSuites.tags.TestsLocal;
+import com.capgemini.mrchecker.core.groupTestCases.testSuites.tags.TestsNONParallel;
+import com.capgemini.mrchecker.selenium.pages.projectY.FileDownloadPage;
+import com.capgemini.mrchecker.selenium.pages.projectY.FileUploadPage;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.io.File;
+
+import static org.junit.Assert.*;
+
+
+@Category({ TestsLocal.class, TestsNONParallel.class })
+public class FileUploadTest extends TheInternetBaseTest {
+
+	private static FileUploadPage fileUploadPage;
+	private static File           downloadedFile;
+
+	@BeforeClass
+	public static void setUpBeforeClass() {
```

Comment:
This should be another test case, not setup for whole test class

Reply:
The intention of setUpBeforeClass method is not to test FileDownloadPage but to utilize FileDownladPage code to get a file for testing. I changed the code a bit to avoid misunderstandings. If you have any suggestions, let me know.